contrib/gitian/README.md

```diff
@@ -233,14 +229,15 @@ This will create a `.sig` file for each `.assert` file above (2 files for each p
 Submitting your signed assert files
 -----------------------------------
 
-Make a pull request (both the `.assert` and `.assert.sig` files) to the
-[monero-project/gitian.sigs](https://github.com/monero-project/gitian.sigs/) repository:
+Fork the [monero-project/gitian.sigs](https://github.com/monero-project/gitian.sigs/) repository and clone the fork. Add both the `.assert` and `.assert.sig` files, as well as your public GPG key (`.asc`) to `gitian-pubkeys` directory. After pushing, create a PR to merge to the upstream.
```

Comment:
You're combining the steps for 1st time submission and subsequent submissions here. The original text is basically correct for subsequent submissions, just omitting the explicit steps of cloning the repo and adding your PGP key.

It would be best if you separate out these initial steps from the subsequent steps, since clearly nobody needs to clone and resubmit PGP key on future submissions.

Reply:
The following **Note:** at line 243 already talks about submitting your PGP key, so not sure it even needs to be stated here.

Reply:
And overall, this isn't the place to teach people how to make pull requests.

Reply:
I will rethink it, but it didn't work out very well here.

> And overall, this isn't the place to teach people how to make pull requests.

Overall I agree, but if a one or two superfluous sentences spare 10 persons the trouble and convinces them not to give up, I think it's worth it.